node/malus/Cargo.toml

```diff
@@ -17,11 +17,13 @@ readme = "README.md"
 publish = false
 
 [dependencies]
-polkadot-cli = { path = "../../cli", default-features = false, features = [ "cli", "malus" ]  }
+polkadot-cli = { path = "../../cli", default-features = false, features = [
+    "cli",
+    "malus",
+] }
 polkadot-node-subsystem = { path = "../subsystem" }
 polkadot-node-subsystem-util = { path = "../subsystem-util" }
 polkadot-node-core-candidate-validation = { path = "../core/candidate-validation" }
-parity-util-mem = { version = "0.10.0", default-features = false, features = ["jemalloc-global"] }
```

Comment:
Please avoid changing this file, https://github.com/paritytech/polkadot/pull/3639 addresses part of it.